utils/scripts/format.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #------------------------------------------------------------------------------
-# © 2021-2022. Triad National Security, LLC. All rights reserved.  This
+# © 2022. Triad National Security, LLC. All rights reserved.  This
```

Comment:
Are we changing the copyright again?

Reply:
oh oops. No this was on me. I copy-pasted this script out of Phoebus.